data/typelibrary/utils-1.0.0/typelib/splitting/WORD_TO_BOOLS.fct

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Function Name="WORD_TO_BOOLS" Comment="">
+	<Identification Standard="61499-1" Description="Copyright (c) 2024 HR Agrartechnik GmbH    &#10;    &#10;This program and the accompanying materials are made     &#10;available under the terms of the Eclipse Public License 2.0     &#10;which is available at https://www.eclipse.org/legal/epl-2.0/     &#10;     &#10;SPDX-License-Identifier: EPL-2.0" >
+	</Identification>
+	<VersionInfo Version="1.0" Author="franz" Date="2024-02-22">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="Normal Execution Request">
+				<With Var="IN"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="Execution Confirmation">
+				<With Var="OUT_00"/>
+				<With Var="OUT_01"/>
+				<With Var="OUT_02"/>
+				<With Var="OUT_03"/>
+				<With Var="OUT_04"/>
+				<With Var="OUT_05"/>
+				<With Var="OUT_06"/>
+				<With Var="OUT_07"/>
+				<With Var="OUT_08"/>
+				<With Var="OUT_09"/>
+				<With Var="OUT_10"/>
+				<With Var="OUT_11"/>
+				<With Var="OUT_12"/>
+				<With Var="OUT_13"/>
+				<With Var="OUT_14"/>
+				<With Var="OUT_15"/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="WORD" Comment="WORD Input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT_00" Type="BOOL" Comment="BIT 00"/>
+			<VarDeclaration Name="OUT_01" Type="BOOL" Comment="BIT 01"/>
+			<VarDeclaration Name="OUT_02" Type="BOOL" Comment="BIT 02"/>
+			<VarDeclaration Name="OUT_03" Type="BOOL" Comment="BIT 03"/>
+			<VarDeclaration Name="OUT_04" Type="BOOL" Comment="BIT 04"/>
+			<VarDeclaration Name="OUT_05" Type="BOOL" Comment="BIT 05"/>
+			<VarDeclaration Name="OUT_06" Type="BOOL" Comment="BIT 06"/>
+			<VarDeclaration Name="OUT_07" Type="BOOL" Comment="BIT 07"/>
+			<VarDeclaration Name="OUT_08" Type="BOOL" Comment="BIT 08"/>
+			<VarDeclaration Name="OUT_09" Type="BOOL" Comment="BIT 09"/>
+			<VarDeclaration Name="OUT_10" Type="BOOL" Comment="BIT 10"/>
+			<VarDeclaration Name="OUT_11" Type="BOOL" Comment="BIT 11"/>
+			<VarDeclaration Name="OUT_12" Type="BOOL" Comment="BIT 12"/>
+			<VarDeclaration Name="OUT_13" Type="BOOL" Comment="BIT 13"/>
+			<VarDeclaration Name="OUT_14" Type="BOOL" Comment="BIT 14"/>
+			<VarDeclaration Name="OUT_15" Type="BOOL" Comment="BIT 15"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION WORD_TO_BOOLS
+
+VAR_INPUT
+	IN : WORD;    // WORD Input
```

Comment:
please remove the comment, as it adds no information

Reply:
@MoritzO02 this is not yet done.